liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-001.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO [RDB].[dbo].[data_sync_config] (table_name, source_db, query, query_with_limit, query_with_null_timestamp)
```

Comment:
We might need to look into how this would work on subsequent deployments after initial deployment. Let me loop @sveselev into this as well.